grouper/fe/templates/email/permission_request_cancelled_html.tmpl

```diff
@@ -0,0 +1,19 @@
+{% extends "email/base_html.tmpl" %}
+
+{% block subject %}Request for Permission Cancelled{% endblock %}
```

Comment:
yeah, i wasn't sure either. i left them separate in case the messaging needs to diverge more (e.g. the current messaging isn't different enough and folks get confused).